README.md

```diff
@@ -166,32 +166,32 @@ git push heroku master
+| [scripts/](https://github.com/factn/resilience-app/tree/master/scripts/)                                         |              These are the node scripts. Again, unless we are changing the way the build works, this shouldn't need to change              |     N |
+| [src/](https://github.com/factn/resilience-app/tree/master/src/)                                                 |                                                     This is where the editing happens.                                                     |     Y |
+| [src/fonts/](https://github.com/factn/resilience-app/tree/master/src/fonts/)                                     |                                                                Local fonts.                                                                |     Y |
+| [src/img/](https://github.com/factn/resilience-app/tree/master/src/img/)                                         |                                          Local image assets for things like the logo and map pins                                          |     Y |
+| [src/js/](https://github.com/factn/resilience-app/tree/master/src/js/)                                           |                                                          All JavaScript and JSX.                                                           |     Y |
+| [src/js/components/](https://github.com/factn/resilience-app/tree/master/src/js/components/)                     |                                              JSX components that are used on different pages                                               |     Y |
+| [src/js/components/inputs/](https://github.com/factn/resilience-app/tree/master/src/js/components/inputs/)       |     JSX components that are also used as input, select, and textarea elements so that the HTML is standardized with much less typing.      |     Y |
+| [src/js/pages/](https://github.com/factn/resilience-app/tree/master/src/js/pages/)                               |                    This folder is for all of the JSX components that wrap full pages and are used as Routes in `App.jsx`                    |     Y |
+| [src/js/resources/](https://github.com/factn/resilience-app/tree/master/src/js/resources/)                       |                            This is general JavaScript that helps the rest of this code do what it needs to do.                             |     Y |
+| [src/scss/](https://github.com/factn/resilience-app/tree/master/)                                                |                                                             All Sass partials                                                              |     Y |
+| [src/scss/global/](https://github.com/factn/resilience-app/tree/master/src/scss/)                                |                                Initial tag settings, set globally before use. Similar to standardized CSS.                                 |     Y |
```

Comment:
Doesn't exist. project went a different route with the styling

Reply:
FWIW I found it helpful to have  a bit of a guide to where things should be when trying to get familiar with things.. 

That said I guess if it's wrong it does more harm than good.

Reply:
I could live with just deleting this whole section I guess...